hookman/hookman_utils.py

```diff
@@ -26,6 +27,8 @@ def load_shared_lib(shared_lib_path: str) -> ctypes.CDLL:
     """
     Load a shared library using ctypes freeing the resource at end.
     """
+    if sys.platform.startswith('win'):
```

Comment:
Please add a comment here why this is necessary

Reply:
We also need a test; I suggest to just ensure that the PATH was changed after loading one of the plugin dlls, no need to have an actual second dll inplace.

Reply:
Also, I added  fixture. Thanks to @williamjamir.